qiskit_nature/operators/second_quantization/star_algebra.py

```diff
@@ -0,0 +1,113 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Sum Operator base interface."""
+
+from abc import ABC, abstractmethod
+
+from .multiply import MultiplyMixin
+# TODO: Move import path after Terra#5617 is merged.
+
+
+class StarAlgebraMixin(MultiplyMixin, ABC):
+    """The star algebra mixin class.
+
+    Star algebra is an algebra with adjoint (dagger).
```

Comment:
```suggestion
    Star algebra is an algebra with an adjoint.
    The :meth:`adjoint` can also be accessed via the `dagger` property for convenience.
```

Is there some special tag for `property` or would one also use `:meth:` there?

Reply:
Thanks. It is pending whether to put the mixin class in the Qiskit docs, so adding the sphinx markup will be in another PR when we need it.